data/data_download/gadm_3.6/Makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: import
+MAKEFLAGS := --jobs=$(shell nproc)
+MAKEFLAGS += --output-sync=target
+
+MarxanUser:=$(shell psql -X -A -t "postgresql://$$API_POSTGRES_USER:$$API_POSTGRES_PASSWORD@marxan-postgresql-api:5432/$$API_POSTGRES_DB" -c "select id from users limit 1")
+
+import: data/gadm/gadm36_simp.geojson
+	ogr2ogr -makevalid \
+		-overwrite \
+		-nln admin_regions -nlt PROMOTE_TO_MULTI \
+		-lco GEOMETRY_NAME=the_geom \
+		-f PostgreSQL PG:"dbname=$$GEO_POSTGRES_DB host='marxan-postgresql-geo-api' \
+		port=5432 user=$$GEO_POSTGRES_USER password=$$GEO_POSTGRES_PASSWORD" $< \
+		-sql "select *,'$(MarxanUser)' as created_by from \"$$(basename -s .geojson "$<")\"";
+
+data/gadm/gadm36_simp.geojson: data/gadm/gadm36_0.shp data/gadm/gadm36_1.shp data/gadm/gadm36_2.shp
```

Comment:
should we add in `data/README.md` a note about this needing plenty of RAM, or even just a comment in this make task?

probably not actually needed since we'll normally be running these scripts in containers that we'll configure via IaC, so we'll set memory and CPU requests in a suitable way.

Reply:
Yes, adding it :D